learn/how-tos/camunda-wavemaker-app-integration.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Add workflow into WaveMaker App by integrating with Camunda BPM"
+id: "camunda-wavemaker-app-integration"
+sidebar_label: "Camunda Integration with WaveMaker App"
+---
+---
+
+## Overview
+
+In this article you will learn how to add workflow into a WaveMaker application by integrating with a opensource BPM product - Camunda. We will use the flexibility of the code generated by WaveMaker and call rules modelled in Camunda.
+
+For demonstration purpose, we will use Employee Leave Management application. The Leave Management application. Employee will apply for the leave using UI and the request goes to the backend where Camunda business rules engine is integrated and based on leave balance, Camunda REST API will return the list of approvers to process the request.
+
+### STEP 1 – CAMUNDA SETUP
+
+1. About Camunda
+  Camunda BPM is a light-weight, open-source platform for Business Process Management. Camunda is a Java-based framework supporting BPMN for workflow and process automation, CMMN for Case Management and DMN for Business Decision Management.
+
+1. Getting Started With Camunda
+   https://docs.camunda.org/get-started/quick-start/
+
+1. [Download and Install Camunda](https://docs.camunda.org/enterprise/download/)
+   Note: For Integrating WM online server, Please install Camunda services on cloud such as AWS/Azure.
+
+1. Create BPMN Process for Apply Leave Request
+   Below is the process flow create in Camunda.
+    [![lftr_sel](/learn/assets/leave_processflow.png)](/learn/assets/leave_processflow.png)
```

Comment:
leave_processflow.png has lot of whitespace on the top. This is how it looks.
![image](https://user-images.githubusercontent.com/57487155/72244993-772ef280-3615-11ea-9ef7-2abea59b6eb4.png)


Can you crop that out?

Reply:
Yes done.